### PR TITLE
cursor move between screens

### PR DESCRIPTION
**Pull request by watasuke102**
## Context

implement a part of #89 

## Summary

The cursor is moved within the screen, and jump to other screens.

## How to check behavior

1. start zen
2. try to move out of screen - it cannot
3. try to move adjacent screen - the cursor is moved to that screen

### CODE REVIEW

File: zen/scene/screen.c [hunk 1]
```diff
@@ -15,6 +15,8 @@ zn_screen_create(
     goto err;
   }
 
+  self->box.width = output->wlr_output->width;
+  self->box.height = output->wlr_output->height;
```
[the remark this answers is not in the document]
Comment (Aki-7) [reply zen/scene/screen.c:19]: And please use `wlr_output_effective_resolution`

**Review by Aki-7 — CHANGES_REQUESTED** [review]

File: zen/cursor.c [hunk 2]
```diff
@@ -22,14 +22,54 @@ zn_cursor_set_screen(struct zn_cursor* self, struct zn_screen* screen)
...
-    self->x = screen->output->wlr_output->width / 2;
-    self->y = screen->output->wlr_output->height / 2;
     wl_signal_add(&screen->events.destroy, &self->destroy_screen_listener);
   }
 
   self->screen = screen;
 }
 
+static void
+zn_cursor_move(struct zn_cursor* self, int x, int y)
```
[the remark this answers is not in the document]
Comment (watasuke102) [reply zen/cursor.c:32]: > Isn't zn_cursor_move_relative enough?
No. This is used by `zn_cursor_move_center()`.

[the remark this answers is not in the document]
Comment (watasuke102) [reply zen/cursor.c:32]: I think naming `zn_cursor_process_move()` looks good.

**Review by Aki-7 — APPROVED** [review]
Thank you!

**Pull request merged**